### DISCUSSION TITLE
Public Document sharing : a document not the whole collection

### DISCUSSION THREAD

**Discussion by IamTaoChen** [outline/outline#5376] [Ideas]
Can we only share a document?

**Comment by tommoor** [comment]
Yes – [see the docs](https://docs.getoutline.com/s/guide/doc/sharing-LG2sGOLIpl).

**Reply by IamTaoChen** [reply]
It's weird. At first, I didn't allow sharing. Then, I turned on the collection's permissions. Now, I can share my collection, but the document cannot be shared. Then I relogin (maybe refresh?), and I can share it.

**Reply by tommoor** [reply]
I believe there is a bug with the permissions being cached locally in this scenario
